config/webpack.dev.js

```diff
@@ -20,11 +20,12 @@ module.exports = merge(
       ...setupWebpackDotenvFilesForEnv({
         directory: RELATIVE_DIRNAME,
         env: MODE
+      }),
```

Comment:
@cdcabrera are we going to worry about cache later ? I know that we want to fix the whole project right now, but do we want to lint only changed files and skip the rest in the near future?

Reply:
@nicolearagao Yes, we do have to migrate the build over to NPM and a few other tidbits to help resolve oddities. 

We're going with this as a low-hanging fruit. It still has to be done so while we're developing (or when we get back to that point) we're receiving linting feedback, see [ds-629](https://issues.redhat.com/browse/DISCOVERY-629)